examples/ios/swift/Migration/README.md

```diff
@@ -0,0 +1,28 @@
+# Migration Examples
+
+The Migration project shows several examples of migrations and migration blocks.
+
+The purpose of this example is to provide a more in depth view of certain problems and pitfalls users might face when migrations get more complex and the number of versions increases.
+
+## How to use the example
+
+You can build and run the project since this is the default state (see `CREATE_EXAMPLES` pre-compiler flag in `AppDelegate`).
+There is nothing much so see in terms of unit tests, console log or in the app itself. If there is no error the migration was successful.
+The files to look at are located in the `Examples` folder. Every file contains an extract of everything necessary for this version (schema version, objects, migration).
+
+How to create new examples:
+1. Uncomment out the `-DCREATE_EXAMPLES` line in `Migration.xcconfig`.
+2. Duplicate `OTHER_SWIFT_FLAGS = $(inherited) -DSCHEMA_VERSION_5;` and increase the version by 1.
+3. Duplicate the most recent `Example_v`x`.swift` file.
+4. Comment out the previous one. Update the current one:
+  * Increase the `schemaVersion`.
+  * Update the `Object` definitions according to your new example.
+  * Update the migration block.
+  * Create some examples for that specific schema version (they will be used in later schema version).
```

Comment:
All of this looks to be in the specified Example_vX.swift file?

Reply:
Correct. I tried to express that by the sub bullet points. Should I clarify further?